patches/resolve-package-path+4.0.3.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/resolve-package-path/lib/index.js b/node_modules/resolve-package-path/lib/index.js
+index a1463f7..a3b5d3c 100644
+--- a/node_modules/resolve-package-path/lib/index.js
++++ b/node_modules/resolve-package-path/lib/index.js
+@@ -89,6 +89,9 @@ function resolvePackagePath(target, baseDir, _cache) {
+             // the custom `pnp` code here can be removed when yarn 1.13 is the
+             // current release. This is due to Yarn 1.13 and resolve interoperating
+             // together seamlessly.
++            if (!pnp) {
++                pnp = require(require.resolve('pnpapi', { paths: [baseDir] }))
```

Comment:
I send the author a PR https://github.com/stefanpenner/resolve-package-path/issues/62

Not sure if he's interested, I think patching this is okay for now. We need to do this, otherwise it won't look in the right directroy for the pnpapi, which is a virtual module dynamically created when you do `require(pnp.cjs).setup()`. Figuring this whole thing out was pretty tedious, I am glad it's done.